time-since-periapsis-and-keplers-equation/universal-variables-example.md

```diff
@@ -49,7 +49,7 @@ $$r_0 = a\frac{e^2 - 1}{1 + e\cos\nu_0}$$
 
 Solving this equation for $e$, we find:
 
-$$0 = e^2 - \frac{r_0}{a}\cos\nu_0 e - \left(a + r_0\right)$$
+$$0 = e^2 - \frac{r_0}{a}\cos\nu_0 e - \left(1 + \frac{r_0}{a}\right)$$
```

Comment:
Thank you, that's 100% correct! I missed `\frac{a + r_0}{a}` which would be equivalent in simplifying.